cmd/lncli/commands.go

```diff
@@ -1788,9 +1788,14 @@ var listChainTxnsCommand = cli.Command{
 		cli.Int64Flag{
 			Name: "end_height",
 			Usage: "the block height until which to list " +
-				"transactions, inclusive, to get transactions " +
-				"until the chain tip, including unconfirmed, " +
-				"set this value to -1",
+				"transactions, inclusive, to get " +
+				"transactions until the chain tip, including " +
+				"unconfirmed, set this value to -1",
+		},
+		cli.StringSliceFlag{
+			Name: "tx_hash",
+			Usage: "return transactions with these transaction " +
+				"hashes",
```

Comment:
Add "can be specified multiple times" as that might not be obvious from the command line help itself.

Reply:
Fixed